src/firmware/guest/types/snp.rs

```diff
@@ -345,13 +377,57 @@ impl Verifiable for (&Chain, &AttestationReport) {
 impl Verifiable for (&Chain, &AttestationReport) {
     type Output = ();
 
+    fn verify(self) -> io::Result<Self::Output> {
+        // According to Chapter 3 of the [Versioned Chip Endorsement Key (VCEK) Certificate and
+        // KDS Interface Specification][spec] and the [Versioned Loaded Endorsement Key Certificate Definition][spec2],
+        // both Versioned Endorsement Key certificates certify an ECDSA public key on curve P-384,
+        // and the signature hash algorithm is sha384.
+        // [spec]: https://www.amd.com/content/dam/amd/en/documents/epyc-technical-docs/specifications/57230.pdf
```

Comment:
I don't think these two links are necessary. Rather, you can mention "see specifications _X_ and _Y_".